### DISCUSSION TITLE
Artillery Cloud Dashboard beta

### DISCUSSION THREAD

**Discussion by hassy** [artilleryio/artillery#1948] [Announcements]
# Artillery Cloud Dashboard

Hello Artillery community! 👋 We're opening up a few more spots in the Artillery Cloud Dashboard beta program and we'd love to have you try it.

## What does it do?

We've designed the dashboard to be the central place for teams using Artillery to visualize, analyze, and collaborate on performance tests with Artillery. Your tests continue running as normal (whether from local, AWS Lambda or AWS Fargate), but you can also configure the reports to be sent to the dashboard.

* See results of your load tests at a glance with `ensure` checks and Apdex scores visualize
* Track load tests as they're running in real time
* Visualize and chart any of the metrics produced by a load test (including custom ones)
* See relevant test metadata, such as region, CPU/memory config, AWS account details if running from AWS Lambda/Fargate etc
* Annotate load test runs with more information for the entire team - results of debugging investigations, links to Datadog dashboards, Jira tickets etc

![CleanShot 2023-06-15 at 10 51 24](https://github.com/artilleryio/artillery/assets/1490/e972afad-3c86-4e90-9199-0ad97ec6eaf9)

# Help us beta test it!

If you'd like early access drop me an email on hveldstra@artillery.io.

We're keeping the beta program pretty small right now, and are looking to work closely with teams that:

* Are actively using Artillery to run load tests frequently (at least a few high-volume load tests every week)
* Don't mind some rough edges
* Happy to provide lots of feedback (we will set up a shared Slack channel)

Looking forward to hearing from you!

**Comment by Thilaknath** [comment]
@hassy  Can i request access for the same ?

**Reply by hassy** [reply]
Sent an invite to the email in your GitHub profile @Thilaknath :)

**Reply by rhiadechavez** [reply]
@hassy 
Can I request access to the Dashboard as well? 
I'm doing a POC for artillery-playwright and I'm interested in exploring the Artillery Dashboard for reporting. 
I would like to assess how we can use it for our future needs. 😄 

Thank you!

**Reply by hassy** [reply]
@rhiadechavez drop me an email on hveldstra@artillery.io and I'll set you up!

**Reply by rhiadechavez** [reply]
Hi @hassy 👋 ,

I just sent an email to you.  😄 
Thank you so much!

**Reply by Thilaknath** [reply]
@hassy I seem to have lost my request. Can you kindly send me the request again to thilaknath@ymail.com

**Reply by hassy** [reply]
@Thilaknath sure thing, just resent the email!

**Comment by navaneethVG** [comment]
We would like to have access to the dashboard, I have requested an email to hveldstra@artillery.io

Navaneeth
Accenture

**Comment by zoe-jobson-IL** [comment]
I have a few test runs, but as the number of users I am using increases so my ability to view the reports tab fails - e.g. https://app.artillery.io/load-tests/t5gg7_bgthptk84rxepbp8jgag578qnqzmr_b9qn its currently not possible to go to the reports tab for this run. I think its because of the sheer number of endpoints triggered by playwright in our test.

I did ask about what I think the cause is (each user has their own socket.io connection with a unique proxy, these requests do not get grouped together)

**Reply by hassy** [reply]
Hi @zoe-jobson-IL 👋 is this related to the issue you reported earlier? https://github.com/artilleryio/artillery/discussions/2440#discussioncomment-8233305

Haven't been able to look at that issue but if it's blocking you from being able to analyze larger test runs we'll prioritize it.

**Reply by zoe-jobson-IL** [reply]
yes that's the one. It does prevent me from using the reports tab with our current load, we were hoping to increase the load significantly over the next week or so. I think its going to block us fairly soon.
